packages/ember-views/lib/views/bound_component_view.js

```diff
@@ -0,0 +1,50 @@
+/**
+@module ember
+@submodule ember-views
+*/
+
+import { _Metamorph } from "ember-views/views/metamorph_view";
+import { read, chain, subscribe, unsubscribe } from "ember-metal/streams/utils";
+import { readComponentFactory } from "ember-views/streams/utils";
+import mergeViewBindings from "ember-htmlbars/system/merge-view-bindings";
+import EmberError from "ember-metal/error";
+
+export default Ember.ContainerView.extend(_Metamorph, {
```

Comment:
@rwjblue I'm open to this approach. I'm curious about what the expected gain is, though. Is it faster or cheaper (in terms of memory)? It seems to me that what we are doing here is very similar what `OutletView` does, which is implemented as `ContainerView.extend(_Metamorph)`.

On that note, I wonder what consideration we should give to extension points for animation (e.g. LiquidFire). @ef4 or others familiar with it?